Networking/RealmManager.swift

```diff
@@ -0,0 +1,126 @@
+                let playerClass = PlayerClass()
+                
+                playerClass.playerID = players[i].id
+                playerClass.name = players[i].name
+                playerClass.height = players[i].height
+                playerClass.date = players[i].date
+                playerClass.weight = players[i].weight
+                playerClass.image = players[i].image
+                playerClass.info = players[i].info
+                playerClass.teamID = players[i].teamID
+                
+                do {
+                    try realm.write {
+                        realm.add(playerClass, update: .all)
+                    }
+                } catch {
+                    print("Failed adding player to a Realm Database")
+                }
+            }
+        }
+    }
+    
+    func addEventsToRealm(events: [Event], teamID: String) {
+        for i in 0..<events.count {
+            if realm.object(ofType: EventClass.self, forPrimaryKey: events[i].eventID) == nil {
+                let eventClass = EventClass()
+                
+                eventClass.eventID = events[i].eventID
+                eventClass.homeTeam = events[i].homeTeam
+                eventClass.awayTeam = events[i].awayTeam
+                eventClass.date = events[i].date
+                eventClass.teamID = teamID
+                
+                do {
+                    try realm.write {
+                        realm.add(eventClass, update: .all)
+                    }
+                } catch {
+                    print("Failed adding event to a Realm Database")
+                }
+            }
+        }
+    }
+    
+    func addTeamsToRealm(teams: [Team]) {
+        for i in 0..<teams.count {
+            if realm.object(ofType: PlayerClass.self, forPrimaryKey: teams[i].teamID) == nil {
+                
+                let teamClass = TeamClass()
+                
+                teamClass.teamID = teams[i].teamID
+                teamClass.name = teams[i].name
+                teamClass.teamImage = teams[i].teamImage
+                teamClass.info = teams[i].info
+                teamClass.stadiumImage = teams[i].stadiumImage
+                
+                do {
+                    try realm.write {
+                        realm.add(teamClass, update: .all)
+                    }
+                } catch {
+                    print("Failed adding team to a Realm Database")
+                }
+            }
+        }
+    }
+    
+    func getModel<T: Object>(selectedType: T.Type) -> [T] {
+        printFileURL()
+        let object = realm.objects(selectedType)
+        var array = [T]()
+        for data in object {
+            array.append(data)
+        }
+        return array
+    }
+    
+    func getPlayers(teamID: String) -> [PlayerClass] {
+        let object = realm.objects(PlayerClass.self).where { $0.teamID == teamID }
+        var array = [PlayerClass]()
```

Comment:
```suggestion
    func getPlayers(teamID: String?) -> [PlayerClass] {
        guard let teamID = teamId else { return [] }
    
        let object = realm.objects(PlayerClass.self).where { $0.teamID == teamID }
        var array = [PlayerClass]()
```
This will make this (and one bellow) method easier to call